admin/build.sh

```diff
@@ -31,7 +31,13 @@ if [[ "$CI_TAG" =~ ^v[0-9]+\.[0-9]+(\.[0-9]+)?(-[A-Za-z0-9-]+)? ]]; then
   ls -l *.sbt project/*.scala
 fi
 
-sbt -Dslick.testkit-config=$TESTKIT_CONF "$publishVersion" ++$SCALA_VERSION testAll updateSampleHelloSlick "root/testSample1" "project root" updateSampleSlickPlainsql "root/testSample2" "project root" updateSampleSlickMultidb "root/testSample3"  "project root" updateSampleSlickTestkitExample "root/testSample4" "project root" $extraTarget
+sbt -Dslick.testkit-config=$TESTKIT_CONF \
+  coverage "$publishVersion" ++$SCALA_VERSION testAll coverageReport \
```

Comment:
Any reason not to do it like this?

```suggestion
  "$publishVersion" ++$SCALA_VERSION \
  coverage testAll coverageReport \
```

That is, run `coverage` after the project version and scala version have been set.

Reply:
the other shouldnt really matter, will apply your suggestion